ironfish/src/rpc/routes/wallet/setAccountHead.ts

```diff
@@ -138,7 +138,8 @@ routes.register<typeof SetAccountHeadRequestSchema, SetAccountHeadResponse>(
 
         const header: BlockHeader | null = await context.chain.getHeader(accountHead.hash)
         Assert.isNotNull(header, 'Account head must be in chain')
-        const transactions = await context.chain.getBlockTransactions(header)
+        const blockTransactions = await context.chain.getBlockTransactions(header)
+        const transactions = blockTransactions.map((txn) => txn.transaction)
```

Comment:
nit -- we talked about this on slack, but we probably don't need to use `getBlockTransactions` here if we only need the transactions from the block

Reply:
I'll open up a new PR to remove usage of this right after.